features/step_definitions/networking.rb

```diff
@@ -1122,4 +1122,40 @@
     step %Q/the outputs should contain "sctp"/
     $i +=1
   end
-end
\ No newline at end of file
+end
+
+Given /^get egress data file directory to the#{OPT_SYM} clipboard$/ do | cb_name |
+  ensure_admin_tagged
+  cb_name = "cb_egress_directory" unless cb_name
+  @result = admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.type}")
+  if @result[:success] then
+     networkType = @result[:response].strip
+  end
+  case networkType
+  when "OVNKubernetes"
+    cb[cb_name]="ovn-egressfirewall"
+  when "OpenShiftSDN"
+    cb[cb_name]="egressnetworkpolicy"
+  else
+    raise "unknown networkType"
+  end
+  logger.info "The egressfirewall fie directory path is stored to the #{cb_name} clipboard."
+end
+
+Given /^get egress type to the#{OPT_SYM} clipboard$/ do | cb_name |
```

Comment:
this step's wording doesn't sound correct to me. Perhap add `I` or `admin` to the beginning

Reply:
Fixed it, thanks!